core/storage/user/gae_models.py

```diff
@@ -2380,20 +2380,18 @@ class UserGroupModel(base_models.BaseModel):
 
     This model stores the user-groups for which feature-flag or
     platform-parameters can be enabled.
-
-    ID for this model is the name of the user-group.
     """
 
     # We use the model id as a key in the Takeout dict.
     ID_IS_USED_AS_TAKEOUT_KEY: Literal[True] = True
 
-    # The user ids of the users.
-    users = datastore_services.StringProperty(repeated=True)
+    name = datastore_services.StringProperty(required=True, indexed=True)
```

Comment:
There should be a comment above each of these lines with the correct docstring.

You might also want to consider calling user_ids below member_ids instead, to match member_names. This is optional though.

Reply:
Updated